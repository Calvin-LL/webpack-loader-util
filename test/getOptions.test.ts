import MyWebpackTestCompiler from "./helpers/MyWebpackTestCompiler";

describe.each([4, 5] as const)("v%d getOptions", (webpackVersion) => {
  it("should work with no options", async () => {
    const compiler = new MyWebpackTestCompiler({ webpackVersion });
    const bundle = await compiler.compile();

    expect(bundle.stats.compilation.warnings).toMatchSnapshot("warnings");
  });
});
