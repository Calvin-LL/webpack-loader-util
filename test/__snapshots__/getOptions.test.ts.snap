// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`v4 getOptions should work with no options: warnings 1`] = `
Array [
  [ModuleWarning: Module Warning (from ./testLoader1.js):
(Emitted value instead of an instance of Error) [object Object]],
]
`;

exports[`v5 getOptions should work with no options: warnings 1`] = `
Array [
  [ModuleWarning: Module Warning (from ./testLoader1.js):
(Emitted value instead of an instance of Error) [object Object]],
]
`;
